opendbc/car/honda/values.py

```diff
@@ -75,7 +76,9 @@ class HondaFlags(IntFlag):
 
   HAS_ALL_DOOR_STATES = 256  # Some Hondas have all door states, others only driver door
   HAS_EPB = 512
+  HAS_HYBRID = 4092
   ALLOW_MANUAL_TRANS = 1024
+  ALLOW_HYBRID = 8192
```

Comment:
You can do away with this.

If you're following the ALLOW_MANUAL_TRANS example, that's just me being extremely conservative about shipping #2457, to flush out any auto-detection failures since failing to the manual trans case would otherwise be non-obvious and take awhile to notice. It can go away once 0.10 has been in release for awhile.

This PR being net-new support, we don't need the extra conservative gating.